packages/ra-ui-materialui/src/layout/Menu.tsx

```diff
@@ -84,14 +105,18 @@ Menu.propTypes = {
     hasDashboard: PropTypes.bool,
 };
 
+// re-export MenuItem commponents for convenience
+Menu.Item = MenuItemLink;
```

Comment:
This is technically a new feature but I think it's really minor, plus it's backwards compatible.